readme.md

```diff
@@ -52,51 +52,52 @@ To install the artifacts in the local maven repository cache `~/.m2/repository`,
 
 #### Building against a custom Flink version
 
-You can check and change the Flink version that Pravega builds against via the `flinkVersion` variable in the `gradle.properties` file.
-Note that you can only choose Flink versions that are compatible with the latest connector code.
+We can check and change the Flink version that Pravega builds against via the `flinkVersion` variable in the `gradle.properties` file.
+
+**Note**: Only Flink versions that are compatible with the latest connector code can be chosen.
 
 #### Building against another Scala version
 
 This section is only relevant if you use [Scala](https://www.scala-lang.org/) in the stream processing application in with Flink and Pravega.
 
-Parts of the Apache Flink use the language or depend on libraries written in Scala. Because Scala is not strictly compatible across versions, there exist different versions of Flink compiled for different Scala versions.
-If you use Scala code in the same application where you use the Apache Flink or the Flink connectors, you typically have to make sure you use a version of Flink that uses the same Scala version as your application.
+Parts of the Apache Flink use the language or depend on libraries written in Scala. Because Scala is **not** strictly compatible across versions, there exist different versions of Flink compiled for different Scala versions.
+If we use Scala code in the same application where we use the Apache Flink or the Flink connectors, we typically have to make sure we use a version of Flink that uses the same Scala version as our application.

```

Comment:
can i rephrase as below?
  Flink  version and Scala version should be same to use the Scala code in the same application where we use Apache Flink or the Flink connectors.